LocalSettings.php

```diff
@@ -1487,6 +1492,12 @@
 			NS_HOENN => 'Hoenn',
 			NS_HOENN_TALK => 'Hoenn_talk',
 		),
+		'studynotekrwiki' => array(
+			'NS_STUDY_NOTE' => 'Study note',
+			'NS_STUDY_NOTE_TALK' => 'Study note_talk',
+			'NS_EXPLANATION' => 'Explanation',
+			'NS_EXPLANATION_TALK' => 'Explanation_talk',
```

Comment:
Please remove the single quotes for the NS_*.
So:
NS_EXPLANATION_TALK

and not(!):
'NS_EXPLANATION_TALK'